### PR TITLE
TM-4444, TM-4437, handle TOD data v3

### PR DESCRIPTION
**Pull request by M00staff**
Tested and working in DEV 

Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2579)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/373)

[Ticket](https://metaphase.atlassian.net/browse/TM-4444)
[Ticket](https://metaphase.atlassian.net/browse/TM-4437)

### CODE REVIEW

File: talentmap_api/fsbid/services/agenda.py [hunk 1]
```diff
@@ -329,9 +329,11 @@ def map_tf(tf=None):
         "org": pydash.get(data, "agendaLegPosition[0].posorgshortdesc", None),
         "eta": pydash.get(data, "ailetadate", None),
         "ted": pydash.get(data, "ailetdtedsepdate", None),
-        "tod": pydash.get(data, "ailtodothertext", None),
-        "tod_months": pydash.get(data, "ailtodmonthsnum", None),
-        "tod_other_text": pydash.get(data, "ailtodothertext", None),
+        "tod": pydash.get(data, "ailtodcode", None),
+        "todMonths": pydash.get(data, "ailtodmonthsnum", None),
+        "todOtherText": pydash.get(data, "ailtodothertext", None),
+        "todShortDesc": pydash.get(data, "todshortdesc", None),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:335]: If we are going to change these ones to be CamelCase, then let's go ahead and change all of them in the payload, to remain consistent

File: talentmap_api/fsbid/services/agenda.py [hunk 2]
```diff
@@ -393,7 +395,11 @@ def fsbid_aia_to_talentmap_aia(data):
         "org": pydash.get(data, "position[0].posorgshortdesc", None),
         "eta": pydash.get(data, "asgdetadate", None),
         "ted": pydash.get(data, "asgdetdteddate", None),
-        "tod": pydash.get(data, 'todshortdesc', None),
+        "tod": pydash.get(data, "asgdtodcode", None),
+        "todMonths": pydash.get(data, "asgdtodmonthsnum", None),
+        "todOtherText": pydash.get(data, "asgdtodothertext", None),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:400]: [same as](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/810/files#r1219744127)

File: talentmap_api/fsbid/services/agenda.py (hunk 1 above)
Comment (M00staff) [reply talentmap_api/fsbid/services/agenda.py:335]: i'll switch it back to snake case!

**Comment by M00staff** [comment]
handled in https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/812

**Pull request closed without merging**